/*
 * Copyright (C) 2017 The Android Open Source Project
 *
 * Licensed under the Apache License, Version 2.0 (the "License");
 * you may not use this file except in compliance with the License.
 * You may obtain a copy of the License at
 *
 *      http://www.apache.org/licenses/LICENSE-2.0
 *
 * Unless required by applicable law or agreed to in writing, software
 * distributed under the License is distributed on an "AS IS" BASIS,
 * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
 * See the License for the specific language governing permissions and
 * limitations under the License.
 */

#ifndef ANDROID_ML_NN_COMMON_HAL_INTERFACES_H
#define ANDROID_ML_NN_COMMON_HAL_INTERFACES_H

#include "vendor/google_paintbox/blue/nn/shared/proto/types.pb.h"

using ::paintbox_nn::Model;
using ::paintbox_nn::Operand;
using ::paintbox_nn::OperandLifeTime;
using ::paintbox_nn::OperandType;
using ::paintbox_nn::Operation;
using ::paintbox_nn::Request;
using ::paintbox_nn::RequestArgument;

#endif // ANDROID_ML_NN_COMMON_HAL_INTERFACES_H
